doc.go

```diff
@@ -21,15 +21,15 @@ It does this by providing stable, idiomatic interfaces for common uses like
 storage and databases. Think `database/sql` for cloud products.
 
 At the core of the Go CDK are common "portable types" implemented by cloud
-providers. For example, objects of the blob.Bucket portable type can be created
-using gcsblob.OpenBucket, s3blob.OpenBucket, or any other provider. Then, the
-blob.Bucket can be used throughout your application without worrying about
-the underlying implementation.
+services. For example, objects of the blob.Bucket portable type can be created
+using gcsblob.OpenBucket, s3blob.OpenBucket, or any other service with a
+Go CDK driver implementation. Then, the blob.Bucket can be used throughout
```

Comment:
And with the change above, this can be simplified to just "or any other driver".

Reply:
Done.